Gemfile.lock

```diff
@@ -67,7 +67,6 @@ GEM
       thread_safe (~> 0.1)
     tzinfo-data (1.2018.5)
       tzinfo (>= 1.0.0)
-    wdm (0.1.1)
```

Comment:
Why has wdm been removed? 🤔

Reply:
i don't know for certain (not a big ruby person) but i think because the docker image runs `bundle install` on linux, and `wdm` is only needed on `x64_mingw`?

Reply:
Ah that sounds very likely! But as we're now not running on Windows, it's not needed - that being said, it may mean that Windows users who don't use Docker may need another `bundle install`

Reply:
Yes, it will need to work in Docker _and_ not in Docker and on Windows

Reply:
yeah i think there's value in leaving it there for people using windows natively. The windows instructions do suggest a bundle install before running jekyll. But presumably this means the lockfile will be a bit... flexible. which is unusual for a lockfile... 🤔